README.md

```diff
@@ -1 +1,157 @@
-# Technology-Innovation-Summit-2023
\ No newline at end of file
+# Technology Innovation Summit 2023
+
+Technology innovation Summit 2023
+
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Technology Innovation Summit 2023] <a name="about-project"></a>
+
+**[Technology Innovation Summit 2023]** is website for a technology conference with the big figures of the industry.
+
+Video description: [@loomhandle](https://www.loom.com/share/e22a99199aa54571950387dfaacfa636)
+
+## 🛠 Built With <a name="built-with"></a>
+
+- <a name="tech-stack">HTML</a>
+
+- <a name="tech-stack">CSS</a>
+
+- <a name="tech-stack">JavaScript</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Innovation Technology Summit 2023]** Show informations about a technology conference that will take place soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Technology Innovation Summit](https://tshobohwa.github.io/Technology-Innovation-Summit-2023/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor
+- Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+- git clone "https://github.com/Tshobohwa/Technology-Innovation-Summit-2023"
+- cd Technology-innovation-summit
+
+### Install
+
+Install this project with:
+
+- cd my-project
+- gem install
+
+### Usage
+
+To run the project, execute the following command:
+
+- live server
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Tshobohwa)
+- Twitter: [@twitterhandle](https://twitter.com/SalomonTshoboh1)
+- LinkedIn: [@linkedIn](https://www.linkedin.com/in/tshobohwa-salomon-4bb457245/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
```

Comment:
- [ ] The previous reviewer asked to remove only the [new_feature_1] in front of your Future feature items. Kindly add your future features back pls.